frontend/src/core/comments/components/Comment.js

```diff
@@ -22,7 +22,7 @@ export default function Comment(props: Props) {
         return null;
     }
 
-    return <li className='comment'>
+    return <li className='comment team-comment'>
```

Comment:
`Comment` is a reusable component, so there should be no code specific to the component that uses it in it. You can simply omit `team-comment` here and use `.comment` in CSS.

Reply:
Forgot about that one 😄 , I originally put that in as I was having trouble getting my margin changes for the Team Comments to work.